CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ Changelog](https://keepachangelog.com/en/1.0.0/).
 - Update `Node` to `20.X`
 - Update `Postgres` to `15.4`
 - Update `Redis` to `7.0.12`
+- Update Node and Yarn install strategy
```

Comment:
- Can you move this line under the main `Changed` section under the other bullet related to `Add required: false...`?
- Can you change the line to "Update Node and Yarn install strategy to remove install script deprecation warning"